hackstar.py

```diff
@@ -336,6 +336,92 @@ def stats():
     return render_template("stats.html", **stats_data)
 
 
+@app.route("/songs")
+def songs():
+    db = get_db()
+    cursor = db.cursor()
+    
+    # Get all songs with their metadata
+    songs_query = """
+        SELECT id, title, artist, release_date, cover
+        FROM song
+        ORDER BY id DESC
+    """
```

Comment:
Would it maybe easier to use a `join` like this instead of several SQL statements/requests?
```sql
select s.*, f.state, f.filename, u.state, u.url
from song s
left outer join job_url u on s.id = u.song_id
left outer join job_file f on s.id = f.song_id
order by s.artist, s.title;
```